### ISSUE TITLE
labsjdk-ce-11-jvmci-21.3-b05 used in GraalVM CE 21.3.0 missing from releases

### ISSUE THREAD

**Issue by zakkak** [graalvm/labs-openjdk-11#19]
Running mx fetch-jdk in graal's tag https://github.com/oracle/graal/releases/tag/vm-21.3.0 results in:

```
Error retrieving https://github.com/graalvm/labs-openjdk-11/releases/download/jvmci-21.3-b05/labsjdk-ce-11.0.13+7-jvmci-21.3-b05-linux-amd64.tar.gz.sha1: HTTP Error 404: Not Found
```

**Comment by dougxc** [comment]
@marwan-hallaoui please take care of this.

**Comment by fniephaus** [comment]
[`jvmci-21.3-b05`](https://github.com/graalvm/labs-openjdk-11/releases/tag/jvmci-21.3-b05) exists but has no assets.

**Comment by marwan-hallaoui** [comment]
Missing artifacts are uploaded and available from [jvmci-21.3-b05](https://github.com/graalvm/labs-openjdk-11/releases/tag/jvmci-21.3-b05).

**Comment by zakkak** [comment]
Thank you @marwan-hallaoui !
